src/languages/r.js

```diff
@@ -128,47 +153,30 @@ export default function(hljs) {
       },
       {
         className: 'number',
+        relevance: 0,
+        beforeMatch: /([^a-zA-Z0-9._])/, // not part of an identifier
         variants: [
           // TODO: replace with negative look-behind when available
           // { begin: /(?<![a-zA-Z0-9._])0[xX][0-9a-fA-F]+\.[0-9a-fA-F]*[pP][+-]?\d+i?/ },
           // { begin: /(?<![a-zA-Z0-9._])0[xX][0-9a-fA-F]+([pP][+-]?\d+)?[Li]?/ },
           // { begin: /(?<![a-zA-Z0-9._])(\d+(\.\d*)?|\.\d+)([eE][+-]?\d+)?[Li]?/ }
```

Comment:
Should this comment be updated? I see `(?<![a-zA-Z0-9._])` being removed down below

Reply:
No, it's moved to `beforeMatch`... if we get look-behind then the commented out code would be what we'd prefer to use.

Reply:
ahhhh I see now, I had gotten the look-behind confused with what you moved to `beforeMatch` and are using in this language extension. me like